src/modules/food/consts/FoodEnum.ts

```diff
@@ -0,0 +1,11 @@
+enum FoodType {
+    All,
+    Dairy,
+    Vegetables,
+    Fruits,
+    Grains,
+    Proteins,
+    Beverages,
+}
+
+export default FoodType;
```

Comment:
Would be great if you can also add a test to make sure no enum types with the same name appear twice. This has caught me in the past during merges.

Reply:
Hi @ageddesi, I have made changes to the test cases and added the test case to ensure all enum types are unique. Please review them when you get a chance :)